svg/global_attributes.json

```diff
@@ -20,7 +20,7 @@
               {
                 "version_added": "78",
                 "partial_implementation": true,
-                "notes": "Only supported on <code>SVGGraphicsElement</code>."
+                "notes": "Only supported on svg <code>&lt;a&gt;</code>, <code>&lt;circle&gt;</code>, <code>&lt;defs&gt;</code>, <code>&lt;ellipse&gt;</code>, <code>&lt;foreignObject&gt;</code>, <code>&lt;g&gt;</code>, <code>&lt;image&gt;</code>, <code>&lt;line&gt;</code>, <code>&lt;path&gt;</code>, <code>&lt;polygon&gt;</code>, <code>&lt;polyline&gt;</code>, <code>&lt;rect&gt;</code>, <code>&lt;svg&gt;</code>, <code>&lt;switch&gt;</code>, <code>&lt;symbol&gt;</code>, <code>&lt;text&gt;</code>, <code>&lt;use&gt;</code> elements."
```

Comment:
This is another excellent example where https://github.com/mdn/browser-compat-data/pull/21495 would come in handy.